index.php

```diff
@@ -0,0 +1,51 @@
+<?php
+?>
+
+<!DOCTYPE HTML>
+<HTML>
+    <head>
+        <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
+        <link rel="stylesheet" type="text/css" href="styles/styles.css" >
+    </head>
+    <body>
+        <h1 id="page title">Your Dream Diary</h1>
+        <h2 id="dreams title">Dreams you have had</h2>
+        <h3 id="dream title">Walls in house made of sponge cake!</h3>
+        <h4 id="dream or nightmare">Dream/Nightmare</h4>
```

Comment:
just wondering why you added id to all these tags. if it's for styling i feel like you could just refer to the <h> and <p> tags directly as they each seem to always apply to the same things

Reply:
sorry: 
just wondering why you added id to all these tags. if it's for styling i feel like you could just refer to the h and
p tags directly as they each seem to always apply to the same things

Reply:
I need to check to see if this is correct for screen readers etc. If it is, I will leave, otherwise I will change attribute type